lib/nlp/nlp_models.yaml

```diff
@@ -0,0 +1,22 @@
+english:
```

Comment:
Maybe place this config in `config/` instead? Will be easier to find and search for.

Reply:
Yeh, I agree. Will rename it as well.